components/automate-chef-io/content/docs/restore.md

```diff
@@ -0,0 +1,96 @@
++++
+title = "Restore"
+description = "Procedures for Restoring Chef Automate Data"
+date = 2018-03-26T15:27:52-07:00
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "get_started"
+    weight = 70
++++
+
+Restore Chef Automate from a [fileystem backup]({{< ref "restore.md#restore-from-a-filesystem-backup" >}}) or an [s3 backup]({{< ref
+"restore.md#restore-from-an-s3-backup" >}}).
+
+## Prerequisites
+
+1. On the restore host, download and unzip the Chef Automate command-line tool:
+
+   ```shell
+        curl https://packages.chef.io/files/current/latest/chef-automate-cli/chef-automate_linux_amd64.zip | gunzip - > chef-automate && chmod +x chef-automate
+    ```
+
+1. **Filesystem backups** require access for Chef Automate to a backup directory in the [configured location]({{< ref
+"backup.md#backup-to-a-filesystem" >}}):
+
+     1. To restore a network-attached filesystem backup, mount the shared backup directory to the same mount point configured at the time of the backup.
+     1. To restore a backup directory that is not a network-attached filesystem, copy the backup directory to the location that was [configured]({{< ref
+"backup.md#backup-to-a-filesystem" >}}) at the time of the backup.
+     1. For restoring a single-file backup archive, copy your archive to the restore host and extract it to the [configured backup directory]({{< ref "backup.md#backup-to-a-filesystem" >}}).
+
+1. **(different FQDN only)** To restore to a host whose FQDN differs from that of the host on which the backup was
+   taken, create a `patch.toml` file to provide at restore time that specifies the new FQDN:
+
+    ```toml
+         [global.v1]
+         fqdn = "<new-fqdn>"
+
+         # To provide a cert and key for the restore host, uncomment and fill
+         # these sections.
+         # [[global.v1.frontend_tls]]
+         # The TLS certificate for the load balancer frontend.
+         # cert = """-----BEGIN CERTIFICATE-----
+         # <certificate-for-new-fqdn>
+         # -----END CERTIFICATE-----
+         # """
+
+         # The TLS RSA key for the load balancer frontend.
+         # key = """-----BEGIN RSA PRIVATE KEY-----
+         # <key-for-new-fqdn>
+         # -----END RSA PRIVATE KEY-----
+         # """
+    ```
+
+## Restore From a Filesystem Backup
+See [prerequisites]({{< ref "restore.md#prerequisites" >}}).
```

Comment:
I did not have a better idea for how to avoid repeating the prerequisites while encouraging people who click through directly to this section to read them; if someone else does, go for it.